clairmeta/dcp_check_subtitle.py

```diff
@@ -721,3 +720,34 @@ def check_subtitle_cpl_image(self, playlist, asset, folder):
                 raise CheckException(
                     "Subtitle image reference {} not found in folder {}"
                     "".format(img, os.path.relpath(folder, self.dcp.path)))
+
+    def check_subtitle_cpl_first_tt_event(self, playlist, asset, folder):
```

Comment:
Should we follow the note from the standard ?

Ie. only perform this check when the position is 1.

> Note: In a multi-reel composition, this only applies to the first reel. This is to accommodate the average buffer size of a wide range of rendering devices.

Reply:
It is not obvious (and I can be wrong), but I interpret it as _the first reel with subtitle essence_.